src/ssd1306-element-element.ts

```diff
@@ -24,6 +56,10 @@ export class Ssd1306ElementElement extends LitElement {
         <!-- 128 x 64 screen -->
         <path fill="#1A1A1A" d="M21 26h100v54H21z" />
 
+        <foreignObject transform="translate(4 6)" x="20" y="20" width="128" height="64">
```

Comment:
Is there a reason we need both `translate` and `x` / `y`?

Reply:
nope I fixed :)